### PR TITLE
Added example of samlp and RP token encryption

### PR DESCRIPTION
**Pull request by nzpcmad**
This example utilizes a strategy with SAMLp and RP token encryption.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,21 @@ passport.use(new wsfedsaml2(
 ));
 ~~~
 
+### Configure strategy for ADFS
```
Comment (siacomuzzi) [inline README.md]: by default ADFS uses WS-Federation, so the title could be confused

maybe `Configure strategy for ADFS (SAML)`?

File: README.md [hunk 2]
```diff
@@ -77,6 +77,21 @@ passport.use(new wsfedsaml2(
...
 
+### Configure strategy for ADFS 
+
+This example utilizes a strategy with SAMLp and RP token encryption.
+
+```javascript
+passport.use('wsfed-saml2', new wsfedsaml2({
+		realm: 'urn:node:samlapp',
+		identityProviderUrl: 'https://my-adfs/adfs/ls',
+        thumbprint: '5D27...D27E',
```
Comment (siacomuzzi) [inline README.md]: `thumbprint: '5D27...D27E', // or options.cert: fs.readFileSync("adfs_signing_key.cer")`

Comment (nzpcmad) [inline README.md]: If ADFS has automatic rollover, there is no way to export the token signing key certificate so no way to generate a .cer file.

Comment (siacomuzzi) [inline README.md]: If you have access to ADFS server:
1) Open ADFS admin tool
2) AD FS -> Service -> Certificates
3) Select `Token-signing` certificate -> View Certificate...
4) Details -> Copy to File...

**Pull request closed without merging**